docs/Using-Fleet/REST-API.md

```diff
@@ -2586,7 +2586,7 @@ Deletes the query specified by name.
 
 #### Example
 
-`DELETE /api/v1/fleet/queries/{name}`
+`DELETE /api/v1/fleet/queries/query_title`
```

Comment:
Dropping the special syntax could lead to users misunderstanding whether this is a dynamic part of the URL.  Is this how it’s being done elsewhere?

Reply:
Yes we do the same for Ids and I like to draw your attention to this being an example as earlier on in the page we covered the general form of that endpoint.